qm-plugins/qm-object-cache/html/class-qm-output-object-cache-slow-ops.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * Output class
+ *
+ * Class QM_Output_Operations_Object_Cache_Slow_Ops
+ */
+class QM_Output_Object_Cache_Slow_Ops extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+
+		global $wp_object_cache;
+		if ( ! method_exists( $wp_object_cache, 'get_stats' ) ) {
+			return;
+		}
+
+		add_filter( 'qm/output/menu_class', array( $this, 'admin_class' ) );
+		add_filter( 'qm/output/panel_menus', array( $this, 'panel_menu' ), 99 );
+	}
+
+	/**
+	 * Outputs data in the footer
+	 */
+	public function output() {
+		$data = $this->collector->get_data();
+		if ( empty( $data ) ) {
+			return;
+		}
+
+		$ops    = $data['slow-ops'] ?? [];
+		$groups = $data['slow-ops-groups'] ?? [];
+
+		$this->before_tabular_output();
+
+		echo '<thead>';
+		echo '<tr>';
+		$this->output_filterable_table_col( __( 'Operation', 'qm-object-cache' ), array_keys( $ops ) );
+		$this->output_sortable_table_col( __( 'Key', 'qm-object-cache' ) );
+		$this->output_sortable_table_col( __( 'Size', 'qm-object-cache' ) );
+		$this->output_sortable_table_col( __( 'Time', 'qm-object-cache' ) );
+		$this->output_filterable_table_col( __( 'Group', 'qm-object-cache' ), $groups );
+		$this->output_sortable_table_col( __( 'Result', 'qm-object-cache' ) );
+		$this->output_sortable_table_col( __( 'Backtrace', 'qm-object-cache' ) );
+		echo '</tr>';
+		echo '</thead>';
+
+		echo '<tbody>';
+		foreach ( $ops as $op_name => $data ) {
+			foreach ( $data as $op ) {
+				echo '<tr data-qm-operation="' . esc_attr( $op_name ) . '" data-qm-group="' . esc_attr( $op['group'] ) . '">';
+				echo '<td class="qm-nowrap qm-ltr">' . esc_html( $op_name ) . '</td>';
+				echo '<td class="qm-nowrap qm-ltr">' . esc_html( $op['key'] ) . '</td>';
```

Comment:
Slow-ops will be able to have an array of keys here as well, so need to support the same type of drop-down thingy as the ops panel. Currently `get_multi()` is excluded from slow_ops, but the new drop-in won't exclude get_multiple().

ATM, this will end up being an array to string conversion / just displaying `Array`.